_posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md

```diff
@@ -0,0 +1,131 @@
+---
+date: 2019-07-16
+title: AWS CloudFormation Example
+categories:
+   - AWS
+description: How to enable CloudFormation and deploy a simple template.
+type: Document
+---
+
+## Background
+
+This document will show you how to enable the CloudFormation stage.  This document also assumes that you have configured an AWS account.  If you need more instructions on configuring your AWS account please refer to either [Deploying to AWS from Spinnaker (using IAM instance roles)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account-iam/) or [Deploying to AWS from Spinnaker (using IAM credentials)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account/) 
+
+## Preconditions
+&nbsp;
+### Enable Cloudformation in Spinnaker
+
+First, we need to enable CloudDriver by adding a `clouddriver-local.yml` file to your hal config profiles directory e.g. `.hal/default/profiles/clouddriver-local.yml`.
+```yaml
+aws:
+  features:
+    cloudFormation:
+      enabled: true
+```
+You can check your configuration by running `hal deploy apply`.
+
+&nbsp;
+### Halconfig 
+
+Make sure to enable the aws provider and have set at least one region
+
+```yaml
```

Comment:
leave this as an example of how it should end up looking like

Reply:
done